Sources/TSCBasic/Path.swift

```diff
@@ -457,6 +458,79 @@ private struct WindowsPath: Path, Sendable {
         return !path.withCString(encodedAs: UTF16.self, PathIsRelativeW)
     }
 
+    /// When this function returns successfully, the same path string will have had the prefix removed,
+    /// if the prefix was present. If no prefix was present, the string will be unchanged.
+    static func stripPrefix(_ path: String) -> String {
```

Comment:
I'd name this Win32PathCchStripPrefix to make it clear that it provides the semantics of that Win32 function; "stripPrefix" is quite general otherwise -- strip _what_ prefix?

Reply:
I don't think that `Win32` is the right prefix, this is not Win32 related. We could use `Zw` as the prefix as per the NT internals.